src/index.js

```diff
@@ -124,13 +123,13 @@ class AppMainRouter extends Component {
     this.setState(newState)
   }
 
-  onAccountChange(account) {
-    this.state.keysManager.miningKeyByVoting(account).then(miningKey => {
-      this.setState({
-        votingKey: account,
-        miningKey: miningKey
-      })
+  async onAccountChange(votingKey) {
+    const miningKey = await this.state.keysManager.miningKeyByVoting(votingKey)
+    this.setState({
+      votingKey,
+      miningKey
     })
+    console.log(`Accounts set:\nvotingKey = ${votingKey}\nminingKey = ${miningKey}`)
```

Comment:
@varasev is this intentional having this log line on production?

Reply:
@vbaranov Yes, I added that to see the addresses in logs for the possible cases if validators report some possible bugs. That logs were in an old DApp's version, then were removed in one of the latest PRs (related to the drop-down network list), and then I returned those back in this PR.